src/client.rs

```diff
@@ -1572,13 +1572,37 @@ pub async fn handle_test_delay(t: TestDelay, peer: &mut Stream) {
     }
 }
 
+/// Whether is track pad scrolling.
+#[inline]
+#[cfg(all(target_os = "macos"))]
+fn check_scroll_on_mac(mask: i32, x: i32, y: i32) -> bool {
+    // flutter version we set mask type bit to 4 when track pad scrolling.
+    if mask & 7 == 4 {
```

Comment:
@xxrlzzz Hi, I'm testing trackpad on win->macOS.

I find trackpad will not work correctly because macOS check if `ControlKey::Scroll` is in the modifiers.

https://github.com/rustdesk/rustdesk/blob/1ededa3476aab868364f6853706e91b7b0ad4e4c/src/server/input_service.rs#LL841C32-L841C32

`check_scroll_on_mac` is only compiled on macOS.

So, I have some questions:

1. Why `check_scroll_on_mac` is needed ? `mask & 7 == 4` is not enough to know whether current event is trackpad event?
2. I cann't understand the logic after `let btn = mask >> 3;`  in `check_scroll_on_mac`. Can you please explain or give a link of what you refered?
   I've revert the code to Commit `260c924`, but I cannot find where `0xff88`, `0x78`, `0x780000` are from.

https://github.com/rustdesk/rustdesk/blob/260c924010361137dd67be145e9d5c0850804768/src/flutter_ffi.rs#L840


`handler.send_mouse` in 
https://github.com/rustdesk/rustdesk/blob/260c924010361137dd67be145e9d5c0850804768/src/ui/remote.tis

Reply:
2个原因
1. 没有其他的设备,所以只支持了mac 的 触控板
2. 当时还在兼容js做ui, hard code是js发出来的,我当时也没有找到这几个hard code的定义,用log打印出来的.

如果现在只有flutter,可以考虑重构一下这里的代码. 重点是识别出触控板和鼠标滚轮的区别, 并正确的发出scroll事件来模拟触控板. 希望能帮到你 :)